eosdis-nasa/earthdata-pub-upload
# Edpub 1093 upload unit testing

# Description

Adding unit test for file upload.

## Spec

See Ticket: [EDPUB-1093](https://bugs.earthdata.nasa.gov/browse/EDPUB-1093)

---

## Validation

1. Pull down the upload branch
2. Run npm run test
3. Confirm that the test passes


---

## Change Log

Adding unit test for file upload.[e5421a857aa14727153caeecd65585cf746d933c](https://github.com/eosdis-nasa/earthdata-pub-upload/commit/e5421a857aa14727153caeecd65585cf746d933c)
